mix/mix_node.nim

```diff
@@ -52,6 +52,7 @@ proc serializeMixNodeInfo*(nodeInfo: MixNodeInfo): Result[seq[byte], string] =
     addrBytes & mixPubKeyBytes & mixPrivKeyBytes & libp2pPubKeyBytes & libp2pPrivKeyBytes
   )
 
+# TODO: set up error enums
```

Comment:
Throughout the code base are error logs. I like the idiom where you return an error enum. it brings with it structured data if need be, and can be assosciated with a string.

This might not be straight forward in `nim` due to it's limited enum construct.